testing/environments/docker/cometd/Dockerfile

```
@@ -1,6 +1,7 @@
 FROM docker.elastic.co/observability/stream:v0.6.1
 
-RUN apt update && \
+RUN sed -i 's/stable/bullseye/g' /etc/apt/sources.list \
```

Would not it need `&&`?

```suggestion
RUN sed -i 's/stable/bullseye/g' /etc/apt/sources.list && \
```